packages/client/src/constants/maps.js

```diff
@@ -0,0 +1,42 @@
+import { ASSET_TYPES, COIN_TYPES, SAFE_TYPES } from "./enums";
+export const ASSET_TYPE_TO_META = {
+  [ASSET_TYPES.TOKEN]: {
+    displayName: "Coins",
```

Comment:
Will here in the future be more properties besides display name?

Reply:
Exactly! It's intend to hold all config data of different types of coins, for example, the url for the thumbnail of the coin